### PR TITLE
French translation of the `Clipboard API` reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates the French translation of the `Clipboard API` reference.

### CODE REVIEW

File: files/fr/web/api/clipboard_api/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: API Clipboard
+slug: Web/API/Clipboard_API
+translation_of: 'Web/API/Clipboard_API'
+---
+<div>{{DefaultAPISidebar("Clipboard API")}}</div>
+
+<p>l'<strong>API Clipboard</strong> (en français : API Presse-papiers) fournit la possibilité de répondre aux commandes du presse-papiers (couper, copier et coller) ainsi que de lire et écrire sur le presse-papiers système de façon asynchrone. L'accès aux contenus du presse-papiers est protégé par l'<a href="/fr/docs/Web/API/Permissions_API">API Permissions</a> : la permission <code>clipboard-write</code> est donnée automatiquement aux pages lorsqu'elles sont dans l'onglet actif. La permission <code>clipboard-read</code> doit quant à elle être demandée, ce que vous pouvez faire en tentant de lire les données du presse-papiers.</p>
```
Comment (SphinxKnight) [inline files/fr/web/api/clipboard_api/index.html:8]: L in uppercase at the beginning :)

Comment (SphinxKnight) [reply files/fr/web/api/clipboard_api/index.html:8]: I wanted to say something about presse-papier vs presse-papiers but hey, seems https://transvision.mozfr.org/?recherche=presse-papier&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings is OK with you and I won't stand a chance :)

File: files/fr/web/api/clipboard_api/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+<div class="notecard note">
+<p><strong>Note :</strong> Cette API <em>n'est pas disponible</em> dans les <a href="/fr/docs/Web/API/Web_Workers_API">Web Workers</a> (elle n'est pas exposée via {{domxref("WorkerNavigator")}}).</p>
+</div>
+
+<p>Cette API est conçue pour remplacer l'accès au presse-papiers précédemment proposé via {{domxref("document.execCommand()")}}.</p>
+
+<h2 id="accessing_the_clipboard">Accéder au presse-papier</h2>
+
+<p>Au lieu de créer un objet <code>Clipboard</code> lors de l'instantiation, vous pouvez accéder au presse-papier du système avec la globale {{domxref("Navigator.clipboard")}} :</p>
```
Comment (SphinxKnight) [inline files/fr/web/api/clipboard_api/index.html:18]: minor typo (detection sponsored by Grammalecte): instantiation > instanciation

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Minor typos to be fixed then this can be merged :)

Thanks a lot @audrasjb

File: files/fr/web/api/clipboard_api/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/api/clipboard_api/index.html]: Ok :D 
The typo is fixed in https://github.com/mdn/translated-content/pull/1770/commits/006b650c77390ce8185d51fb09d2f3c0345f4bd2

File: files/fr/web/api/clipboard_api/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/api/clipboard_api/index.html]: thanks, typo fixed in https://github.com/mdn/translated-content/pull/1770/commits/006b650c77390ce8185d51fb09d2f3c0345f4bd2

**Review by SphinxKnight — APPROVED** [review]

**Pull request merged**